libraries/core-react/stories/components/Banner.stories.tsx

```diff
@@ -11,105 +11,113 @@ const icons = {
   mood_sad,
 }
 
-const { BannerIcon, BannerMessage, BannerActions } = Banner
-Icon.add(icons)
+const { Icon, Message, Actions } = Banner
```

Comment:
Remove this desconstruct and remove rename of  `Icon` -> `EdsIcon`?

Reply:
Some syntax fiddling gives this as an option, I suggest we use that approach 🤷‍♀️ 

```
subcomponents: {
    Message: Banner.Message,
    Icon: Banner.Icon,
    Actions: Banner.Actions,
  },
```